docs/nextsteps.rst

```diff
@@ -3,65 +3,71 @@
 Next Steps
 ==========
 
+Once you have rendered your package and set it up on `GitHub
+<https://github.com>`__ you may wish to enable `Travis CI
+<https://travis-ci.org>`_ , `Appveyor <https://www.appveyor.com/>`_ and
+`Read the Docs <https://readthedocs.org/>`_.  Configuration for these services
+is included in the template, and while their use is optional, it is
+recommended.
+
 Setting Up Continuous Integration
 ---------------------------------
 
-Once you have rendered your package and set it up on `GitHub
-<https://github.com>`__ you may wish to enable Travis CI, Appveyor and Read the
-Docs (configuration for these services is included in the template, but you
-could utilise others).
+`Continuous Integration 
+<https://github.com/astropy/astropy/wiki/Continuous-Integration>`__ (CI)
```

Comment:
this is a very obsolate wiki page, what about linking it to wikipedia instead (and maybe to the https://en.wikipedia.org/wiki/Continuous_testing page as the integration seems slightly different, or actually the travis docs really nice and brief yet informative on this: https://docs.travis-ci.com/user/for-beginners/#What-is-Continuous-Integration-(CI)%3F).